.env

```diff
@@ -1,3 +1,15 @@
+### Docker settings
+HOST_IP=docker.for.mac.host.internal
+
+# MySQL database credentials
+MYSQL_ROOT_PASSWORD=Admin123
+MYSQL_DATABASE=phpminds
+MYSQL_USER=web_user
+MYSQL_PASSWORD=User123
```

Comment:
:stop_sign: I thought `.env` is meant to not be committed anywhere?

Reply:
Traditionally you're not supposed to. That's right.
But in Symfony 5 `.env` is the default file that can be overridden by `.env.<environment>`